src/core/core/rpc_engine.cpp

```diff
@@ -144,7 +144,12 @@ namespace dsn {
                 req->server_address.group_address()->leader_forward();
             }
 
-            call->set_delay(delay_ms);
+            // the "delay_ms" is simulator's net_delay_milliseconds,
+            // the "call->delay_milliseconds()" is user specified delay time to dispatch response task,
```

Comment:
I think sum is more proper. When upper apps pecified the delay, they mean to delay dispatch the response task. And the delay_ms is for simulator to simulate network delay. So the sum time is more correct with expect result: network_delay + dispatch_delay.